boards/openmote-b/doc.txt

```diff
@@ -0,0 +1,59 @@
+/**
+@defgroup    boards_openmote-b OpenMote-b
+@ingroup     boards
+@brief       Support for the OpenMote-b board
+
+## Overview
+
+The [OpenMote](http://www.openmote.com/) is slim board that comes with a TI
+SoC combining an ARM Cortex-M3 microcontroller with an IEEE802.15.4 radio.
+
+## Hardware
+| MCU        | CC2538SF53        |
+|:------------- |:--------------------- |
+| Family | ARM Cortex-M3     |
+| Vendor | Texas Instruments |
+| RAM        | 32Kb  |
+| Flash      | 512Kb             |
+| Frequency  | 32MHz |
+| FPU        | no                |
+| Timers | 4 |
+| ADCs       | 1x 12-bit (8 channels)        |
+| UARTs      | 2                 |
+| SPIs       | 2                 |
+| I2Cs       | 1                 |
+| Vcc        | 2V - 3.6V         |
+| Datasheet  | [Datasheet](http://www.ti.com/lit/gpn/cc2538) (pdf file) |
+| Reference Manual | [Reference Manual](http://www.ti.com/lit/pdf/swru319) |
+
+##  Flashing and Debugging
+
+Currently RIOT supports flashing the OpenMote using a Segger JLink JTAG
+adapter or via USB, using the bootloader on the board.
+
+### Flashing via USB
+
+RIOT support flashing with USB by default.
+
+`make flash`
+
+You may have to specify the flashing port with
+`PORT_BSL=<my_openmote_port> make flash`
+
+Please note that on some boards the MSP430 chip responsible for controlling
```

Comment:
could wrap this into an `@note` doxygen tag

Reply:
Noted :wink: